entrypoint.py

```diff
@@ -336,6 +341,12 @@ def _copy_java_validators():
         default="",
         action="store_true",
     )
+    parser.add_argument(
```

Comment:
this solution is ok for a maven specific script.
In this case, adding a parameter for each specific tool (maven, trivy, ...) will make the tool unmaintainable in time.